Formula/n/netatalk.rb

```diff
@@ -52,17 +58,21 @@ class Netatalk < Formula
 
   conflicts_with "ad", because: "both install `ad` binaries"
 
+  resource "bstring" do
+    url "https://github.com/msteinert/bstring/releases/download/v1.0.1/bstring-1.0.1.tar.xz"
+    sha256 "a86b6b30f4ad2496784cc7f53eb449c994178b516935384c6707f381b9fe6056"
+  end
```

Comment:
We can replace this with
```ruby
depends_on "bstring"
```
after #232491.

Reply:
Should be good to go once https://github.com/Homebrew/homebrew-core/pull/232506 is merged.